src/test/java/graphs/GraphSearchTest.java

```diff
@@ -469,18 +476,18 @@ public void setUp() throws Exception {
          *
          */
         List<GraphEdge> edgesForGraph11 = new LinkedList<GraphEdge>();
-        edgesForGraph11.add(edgeBetweenAandD.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenZandD.setWeighted(true).setWeight(2));
-        edgesForGraph11.add(edgeBetweenBandD.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenHandZ.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenBandH.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenCandE.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenGandD.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenZandG.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenFandB.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenDandC.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenEandD.setWeighted(true).setWeight(1));
-        edgesForGraph11.add(edgeBetweenDandF.setWeighted(true).setWeight(1));
+        edgesForGraph11.add(edgeBetweenAandD.setWeight(1));
```

Comment:
why not set the weight on the edge in a constructor? or even better, declare this graph unweighted?